homeassistant/components/habitica/sensor.py

```diff
@@ -39,6 +41,78 @@
     "class": SensorType("Class", "mdi:sword", None, ["stats", "class"]),
 }
 
+
+@dataclass(kw_only=True, frozen=True)
+class HabitipySensorEntityDescription(SensorEntityDescription):
+    """Habitipy Sensor Description."""
+
+    value_path: list[str]
+
+
+SENSOR_DESCRIPTIONS: dict[str, HabitipySensorEntityDescription] = {
+    "name": HabitipySensorEntityDescription(
+        key="name",
+        translation_key="name",
+        value_path=["profile", "name"],
+    ),
+    "hp": HabitipySensorEntityDescription(
+        key="hp",
+        translation_key="hp",
+        native_unit_of_measurement="HP",
+        suggested_display_precision=0,
+        value_path=["stats", "hp"],
+    ),
+    "maxHealth": HabitipySensorEntityDescription(
+        key="maxHealth",
+        translation_key="maxhealth",
+        native_unit_of_measurement="HP",
+        value_path=["stats", "maxHealth"],
+    ),
+    "mp": HabitipySensorEntityDescription(
+        key="mp",
+        translation_key="mp",
```

Comment:
What's MP?

Reply:
Mana Points. A lot of things i just took over from the code that was already there, didn't want to change too much in a single PR, but if you prefer it i could try to improve more things, especially for better readability..

Reply:
I mean let's rename them to something that works well

Reply:
already on it ^^